packages/twenty-server/src/engine/integrations/custom-code-engine/utils/temporary-lambda-folder-manager.ts

```diff
@@ -0,0 +1,45 @@
+import { join } from 'path';
+import { tmpdir } from 'os';
+import fs from 'fs';
+
+import fsExtra from 'fs-extra';
+import { v4 } from 'uuid';
+
+const TEMPORARY_LAMBDA_FOLDER = 'twenty-build-lambda-temp-folder';
+const TEMPORARY_LAMBDA_SOURCE_FOLDER = 'src';
+const LAMBDA_ZIP_FILE_NAME = 'lambda.zip';
+const LAMBDA_ENTRY_FILE_NAME = 'index.js';
+
+export class TemporaryLambdaFolderManager {
```

Comment:
The naming makes it feel like it's a hack/temporary file that will need to be refactored/go away, is that the case? Didn't go in-depth yet but I have a feeling it should be renamed if not

Reply:
maybe temporaryDir = buildDir?

Reply:
yes it is a buildDirManager, i will rename